### DISCUSSION TITLE
Specific OTF bitmap font fails to load

### DISCUSSION THREAD

**Discussion by iFiz** [ImageMagick/ImageMagick#6688] [Contribute]
### ImageMagick version

7.1.0-29

### Operating system

MacOS

### Operating system, version and so on

macOS Monterey 12.3

### Description

Hi! We've having some issues with reading specific .OTF fonts which contains bitmaps. This is most easily reproduced using the `convert` command in the CLI. ImageMagick returns a generic error, which makes it hard to debug if the issue is with the font or with ImageMagick. The font works fine in the browser / Font Book, etc. 

`convert -list format | grep OTF` returns `OTF* TTF       r--   Open Type font (Freetype 2.11.1)`

Could this be an issue with the font or perhaps an issue with configuration / ImageMagick? It would be great if anyone could assist with this. Thanks!

### Steps to Reproduce

- Download the font (https://static.yoursurprise.com/resourcebase/font/2701.otf)
- Install the font 
- Run `convert -background white -font Chocotelegram-70px -size 165x70 -pointsize 24 -debug annotate label:Test  output.jpg`
- The following error is thrown: `convert: unable to read font `/Users/{user}/Library/Fonts/2701.otf' @ error/annotate.c/RenderFreetype/1560.`

**Comment by urban-warrior** [comment]
ImageMagick supports OTF fonts, but not this particular type.  It will require a bit of work to support it.  We currently do not have an ETA.

**Comment by dlemstra** [comment]
I just added something to improve the error message. This might help someone who is willing to pick up this issue:
```
I:\issues\im7\4997>magick -background white -font 2701.otf -size 165x70 -pointsize 24 label:Test output.jpg
magick: unable to read font `2701.otf (invalid pixel size)' @ error/annotate.c/RenderFreetype/1575.
```
